package.json

```diff
@@ -102,23 +105,30 @@
       "^(?!.*\\.(js|jsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
     },
     "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$"
+      "node_modules/?!(react-data-grid|mammoth)"
     ],
     "moduleDirectories": [
       "src",
       "node_modules"
     ],
-    "setupTestFrameworkScriptFile": "<rootDir>/tests/setupTests.js"
+    "setupFilesAfterEnv": [
+      "<rootDir>/tests/setupTests.js"
+    ]
   },
-  "eslintConfig": {
-    "extends": "airbnb",
-    "plugins": [
-      "import"
-    ],
-    "settings": {
-      "import/resolver": {
-        "webpack": "webpack.config.js"
-      }
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
     }
-  }
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx,ts,tsx,json,md}": [
```

Comment:
Markdown files might be outside of the `src/` directory I think we'll still want those linted as well.

Reply:
We probably want to include the tests folder to match as well, or were you trying to avoid a lot of fixes?